autopts/wid/gap.py

```diff
@@ -1708,7 +1708,7 @@ def hdl_wid_301(params: WIDParams):
 
     # GAP/PADV/PASE/BV-02-C starts PA Sync in WID 302 which comes first
     if params.test_case_name not in ['GAP/PADV/PASE/BV-02-C']:
-        btp.gap_padv_create_sync(0, 0, 10, 0)
+        btp.gap_padv_create_sync(0, 0, 500, 0)
```

Comment:
As I also mentioned on Discord: The appropriate way of handling this would be to apply a function to the PA interval we received. While unlikely, this may still fail if PTS changes the PA interval to something larger (is the interval PTS uses an IXIT value?)